packages/editor/package.json

```diff
@@ -75,8 +75,8 @@
   },
   "dependencies": {
     "@inquirer/core": "^10.1.15",
-    "@inquirer/type": "^3.0.8",
-    "external-editor": "^3.1.0"
+    "@inquirer/external-editor": "workspace:*",
```

Comment:
Will need a version so the publish works; not sure if Lerna replaces that transparently 🤷🏻‍♂️